fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -33,13 +33,21 @@ def default_objective(self):
         """Return the default objective for moments of this kind."""
         return ErrorRate()
 
-    def load_data(self, X, y, event=None, **kwargs):
+    def load_data(self, X, y, event=None, filter_value=None, **kwargs):
         """Load the specified data into this object.
```

Comment:
I think that the following proposal will be better:
* let's get rid of the `filter_value` field
* if `event` contains `numpy.nan` those values should be excluded from conditioning

This will be consistent with the `pandas.DataFrame.groupby` and may also lead to a more concise code; e.g.:
```
>>> import pandas as pd
>>> import numpy as np
>>> df = pd.DataFrame({"event": ['a', 'a', np.nan, 'b'], "val": [4, 3, 2, 1]})
>>> df.groupby("event").size()
event
a    2
b    1
dtype: int64
```

Reply:
So where do you set it to np.nan?